tools/seed/main.go

```diff
@@ -413,6 +417,43 @@ func generateCodesAndStats(db *database.Database, realm *database.Realm) error {
 	return nil
 }
 
+// generateKeyServerStats generates stats normally gathered from a key-server. This is
+// primarily used to test statistics and graphs.
+func generateKeyServerStats(db *database.Database, realm *database.Realm) error {
+	if err := db.SaveKeyServerStats(&database.KeyServerStats{RealmID: realm.ID}); err != nil {
+		return fmt.Errorf("failed create stats config: %w", err)
+	}
+
+	nowish := time.Now().Add(-5 * time.Minute)
+	for day := 0; day < 30; day++ {
+		date := nowish.Add(time.Duration(day) * -24 * time.Hour)
```

Comment:
I think we should UTC midnight this (or the one on 427)

Reply:
UTCed - not sure if midnight really helps

Reply:
in production, they'll be UTC midnight for days

Reply:
that's what you'll get from the key server